docs/application/native/guides/multimedia/overview.md

```diff
@@ -39,7 +39,7 @@ You can use the following multimedia features in your native applications:
 
 - [Sound Manager](sound-manager.md)
 
-  You can control the output volume of a specific volume type (aka sound type). You can define which stream type is used for a playback or recording handle in your application. You can manage sound conflict senarios by using stream focus. You can also query for various information related to the connected sound devices, for example, type, name, id, direction and state.
+  You can control the output volume of a specific volume type (aka sound type). You can define which stream type is used for a playback or recording handle in your application. You can manage sound conflict scenarios by using stream focus. You can also query for various information related to the connected sound devices, for example, type, name, id, direction and state.
```

Comment:
Consider adding a comma to separate the elements.
```suggestion
  You can control the output volume of a specific volume type (aka sound type). You can define which stream type is used for a playback or recording handle in your application. You can manage sound conflict scenarios by using stream focus. You can also query for various information related to the connected sound devices, for example, type, name, id, direction, and state.
```

Reply:
I'll keep in mind these comments not related to webrtc, I'll update it with another PR in the near future. Thank you.